tests/infrared/README.md

```diff
@@ -17,7 +17,7 @@ to an STF instance all on one (large) baremetal machine.
 Once the deployment is complete, you can check prometheus for data, like so:
 
 ```shells
-$ PROM_HOST=$(oc get route prometheus -o jsonpath='{.spec.host}')
+$ PROM_HOST=$(oc get route default-prometheus-proxy -o jsonpath='{.spec.host}')
```

Comment:
This is not actually a result of the change, right? Just something you spotted?

Reply:
Yea just something spotted as I went through all the files.